internal/core/src/segcore/InsertRecord.h

```diff
@@ -274,7 +274,7 @@ class OffsetOrderedArray : public OffsetMap {
                 hit_num++;
             }
         }
-        return seg_offsets;
+        return {seg_offsets, it != array_.end()};
     }
```

Comment:
here, it may not be very accurate because when bitset.count() is 0, it will not move to the end of the array neither. But has_more_result is a 'possible and vague' tag as explained before, so it's ok to simplify here like this